# def errors(func):
#     def new_handler(*args, **kwargs):
#         try:
#             result = func(*args, **kwargs)
#         except:
#             result = 0
#             print('Сработал обработчик ошибок!')
#
#         return result
#
#     return new_handler

